package.json

```diff
@@ -10,7 +10,7 @@
 		"watch": "concurrently \"npm run server\" \"npm run frontend\"",
 		"start": "node ./build",
 		"heroku-postbuild": "bash ./production-build.sh",
-		"test": "tsc **/test/*.ts && jest && rm **/test/*.js",
+		"test": "jest",
```

Comment:
Still get errors:

$ jest
● Validation Error:

  Preset ts-jest not found.

  Configuration Documentation:
  https://jestjs.io/docs/configuration.html

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

No ts-jest in package.json?

Reply:
Hmm I had to make a new branch because I screwed some other stuff up so I think I accidentally missed the package.json. Should be fixed now! Thanks!